showItemPageBE/src/main/java/com/talentpath/shamazin/showItemPage/models/Question.java

```diff
@@ -21,6 +25,14 @@
 
     private LocalDate date;     // Store the date of when the question was asked
 
+    @JsonIgnore
+    @ManyToOne(fetch=FetchType.LAZY)
+    @NotBlank
+    private ItemFamily itemFamily;
+
+    @OneToMany(mappedBy = "answer", cascade=CascadeType.ALL)
```

Comment:
mappedBy="question" -- mappedBy is what we use to tell Hibernate which variable we are using to represent parent class in our child class =)